app/views/admin/invoices/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Invoice #<%= @invoice.id %></h1>
+
+<%= form_with url: "/admin/invoices/#{@invoice.id}", method: :patch, data: { turbo: false } do |form| %>
+  <%= form.label :status, "Status:" %>
+  <%= form.select :status, ["cancelled", "completed", "in progress"], selected: @invoice.status %>
```

Comment:
I don't fully understand what's happening here, but I trust that you have it right! I also would really like to pick your brain about what's happening under the hood on this